### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by KatBrandt**
## Issue ticket number and link
Landing Page Story Card

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] I have added thorough tests.

## Notes
Completes user story: Landing Page

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
```
Comment (dcoleman21) [inline app/models/user.rb:1]: Let's not forget to set up these associations and validations

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```
Comment (dcoleman21) [inline spec/models/user_spec.rb:3]: Let's not forget to add these associations/validation tests

**Review by dcoleman21 — COMMENTED** [review]
This all looks great. I think we just need to add model associations/validations for the User model/spec

File: app/models/user.rb (hunk 1 above)
Comment (KatBrandt) [reply app/models/user.rb:1]: Good call.

**Pull request merged**